code/game/objects/items/rogueweapons/melee/blunt.dm

```diff
@@ -72,15 +72,15 @@
 	wdefense = 3
 
 /obj/item/rogueweapon/mace/steel
-	force = 30
-	force_wielded = 40
+	force = 25
+	force_wielded = 35
 	name = "steel mace"
 	desc = "This steel mace is objectively superior to an iron one."
 	icon_state = "smace"
 	wbalance = -1
 	smeltresult = /obj/item/ingot/steel
 	blade_dulling = DULLING_BASH
-	wdefense = 3
```

Comment:
I think the wdefense should stay at 3, rn its less than the iron at 2

There should be reasons to upgrade outside of damage

Reply:
yeah

Reply:
> I think the wdefense should stay at 3, rn its less than the iron at 2
> 
> There should be reasons to upgrade outside of damage

Iron maces have 2 wdef